rest/kadai-rest-spring-example-boot/src/main/java/io/kadai/example/boot/security/BootWebSecurityConfigurer.java

```diff
@@ -170,8 +153,8 @@ public LdapAuthoritiesPopulator authoritiesPopulator(
     return populator;
   }
 
-  @Bean
-  public DefaultSpringSecurityContextSource defaultSpringSecurityContextSource() {
+  @Bean(name = KADAI_LDAP_CONTEXT_SOURCE)
+  public BaseLdapPathContextSource ldapContextSource() {
```

Comment:
Why BaseLdap "PATH" ContextSource? Is path a important detail here?

Reply:
BaseLdapPathContextSource is the most general interface that can be created and injected as bean in BootWebSecurityConfigurer. It is just called in this way